src/Components/DeleteComment.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import { deleteComments } from "../utility/api";
+import { useParams } from "react-router-dom";
+const DeleteComment = ({comment_id, article_id, setComments}) =>{
+    const [isDeleting, setIsDeleting] = useState(false)
+    const [userComment, setUserComments] = useState('');
+    const [comment, setComment] = useState('');
+    const handleDelete = () => {
+        setIsDeleting(true)
+            deleteComments(comment_id)
+          .then((newComments) => {
+            setComments((prevComments) =>
+                prevComments.filter((comment) => comment.comment_id !== comment_id)
+              );
+            setIsDeleting(false);
+          })
+              .catch((err)=>{
+                console.err("we broke:", err)
```

Comment:
Remember we only need one catch in a promise chain and you already have a catch on the API function. As such this catch block will not not run. But we need it to - to feedback to the user if the delete was unsuccessful (eg no network connection). Also I think you mean `console.error()` 😊